core/src/kura.rs

```diff
@@ -194,6 +205,17 @@ impl Kura {
                         iroha_logger::telemetry!(msg = iroha_telemetry::msg::SYSTEM_CONNECTED, genesis_hash = %new_block.hash());
                     }
 
+                    if let Some(path) = kura.block_plain_text_path.as_ref() {
+                        let mut plain_text_file = std::fs::OpenOptions::new()
+                            .create(true)
+                            .append(true)
+                            .open(path)
+                            .expect("Couldn't create file for plain text blocks.");
+
+                        serde_json::to_writer_pretty(&mut plain_text_file, &new_block)
```

Comment:
I want to notice one thing: despite this file called `blocks.json` it is not valid json file, because it has the following structure:
```
{ BLOCK 1}
{ BLOCK 2}
{ ...... }
{ BLOCK N}
```

However converting it to valid `json` is rather straightforward postprocessing: 
```
[
{ BLOCK 1},
{ BLOCK 2},
{ ...... },
{ BLOCK N}
]
```